src/a11y-dialog.ts

```diff
@@ -209,12 +204,8 @@ export default class A11yDialog {
    * See: https://github.com/KittyGiraudel/a11y-dialog/issues/177
    */
   private maintainFocus = (event: FocusEvent) => {
-    if (!this.shown) return
```

Comment:
🔍 Same as `bindKeypress` – checking for `shown` is redundant, right?

Reply:
I guess so yes.